src/components/adaptive-cards/Container/Container.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+import Component, {registerComponent} from '../Component/Component';
+
+/**
+ * Adaptive Cards Container component
+ * https://adaptivecards.io/explorer/Container.html
+ *
+ * @param {object} props  React props passed to the component
+ * @param {object} props.data  Active cards definition
+ * @returns {object} JSX of the component
+ */
+export default function Container({data}) {
+  const [, sc] = webexComponentClasses('container');
+  const classes = [];
+
+  const getClass = (key, value) => sc(`${key}--${value.toLowerCase()}`);
```

Comment:
careful, value can be different than string, for example a boolean (bleed=true)

true.toLowerCase() gives an error

let's handle the case issue in a different task